evm/vm/forks/__init__.py

```diff
@@ -13,3 +13,11 @@
 from .byzantium import (  # noqa: F401
     ByzantiumVM,
 )
+
+ALL_VM = [
+    TangerineWhistleVM,
+    FrontierVM,
+    HomesteadVM,
+    SpuriousDragonVM,
+    ByzantiumVM
```

Comment:
A potentially better way to get this list would be to extract them from the`MainnetChain`

Reply:
:+1: as in `ALL_VM = [vm for _, vm in MainnetChain.vm_configuration]` I guess